field_friend/automations/field_parameters.py

```diff
@@ -0,0 +1,94 @@
+import math
+from dataclasses import dataclass
+from typing import Any
+from uuid import uuid4
+
+from rosys.geometry import Point
+from shapely import offset_curve
+from shapely.geometry import LineString, Polygon
+from typing_extensions import Self
+
+from field_friend.localization import GeoPoint
+
+from . import Field, Row
+
+
+class FieldParameters():
```

Comment:
The parentheses are only needed when the class extends another
```suggestion
class FieldParameters:
```